app/services/concerns/xccdf/rule_groups.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module Xccdf
+  # Methods related to saving groups
+  module RuleGroups
+    extend ActiveSupport::Concern
+
+    included do
+      def save_rule_groups
+        @rule_groups ||= @op_rule_groups.map do |op_rule_group|
+          ::RuleGroup.from_openscap_parser(op_rule_group, benchmark_id: @benchmark&.id)
+        end
+
+        ::RuleGroup.import!(new_rule_groups, ignore: true)
+      end
+
+      def save_parent_rule_groups
+        @op_rule_groups.each do |op_rule_group|
+          rule_group_for(ref_id: op_rule_group.id).update!(parent: rule_group_for(ref_id: op_rule_group.parent_id))
+        end
+      end
+
+      private
+
+      def split_rule_groups
+        @split_rule_groups ||= @rule_groups.partition(&:new_record?)
+      end
+
+      def old_rule_groups
```

Comment:
Is this used anywhere?

Reply:
Hmm.. no that was copied from `rules.rb` and `profiles.rb` but I'll follow how `rules.rb` after you updated it in [this](https://github.com/RedHatInsights/compliance-backend/pull/1151/files) PR

Reply:
Yup, that's some legacy stuff remained after I removed the upstream profile/rule adjustments.